api/app/tests/Behat/bootstrap/v2/Registration/SelfRegistrationTrait.php

```diff
@@ -153,6 +157,21 @@ public function iShouldSeeAnInvalidClientLastnameError()
         $this->assertOnErrorMessage($this->invalidClientLastnameError);
     }
 
+    /**
+     * @Then /^an incorrect case number length error is \'([^\']*)\'$/
+     */
+    public function anIncorrectCaseNumberLengthErrorIs($arg1)
+    {
+        if ('not thrown' == $arg1) {
+            $this->assertPageContainsText(sprintf("We've sent you a link to %s that you need to click to activate your deputy service account.", $this->userEmail));
+        } else {
+            $actualErrorMessage = $this->getSession()->getPage()->find('css', 'ul.govuk-list.govuk-error-summary__list')->getHtml();
+            $actualErrorMessageStripped = strip_tags($actualErrorMessage);
+
+            $this->assertStringEqualsString($actualErrorMessageStripped, $this->incorrectCaseNumberLengthError, 'incorrect case number length error thrown');
```

Comment:
I would probably change the final argument to "Incorrect case number length error was expected but not seen", to make it clear that we're expecting to see something but we're not.